platform/android/README.md

```diff
@@ -66,6 +90,13 @@ $ sudo apt-get install -y build-essential curl lib32stdc++6 lib32z1 pkg-config p
 
 #### Open project in Android Studio
 
```

Comment:
On macos, you can execute 'make aproj' this will automatically open the project in AS

Reply:
'make aproj' instructions added